src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStream.cs

```diff
@@ -6,6 +6,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Data.SqlClient.Microsoft.Data.SqlClientX.IO;
```

Comment:
Please fix namespace to "Microsoft.Data.SqlClientX.IO"

Reply:
Will do. Visual Studio creates this default NS based on the assembly name and the folder of the class. 
I will pay more attention to it.